countdown.js

```diff
@@ -1,5 +1,14 @@
 function countdown(seconds){
-  // ...
+	var tick = 0;
+
+	setInterval(function() {
+        if (seconds > tick) {
+        	seconds--;
+        	console.log(seconds);
+        }
```

Comment:
Well, it stops printing numbers, but the interval continues to run. Can you make it stop once it's finished the countdown?